src/wfs/src/test/java/org/geoserver/wfs/WFSServiceExceptionTest.java

```diff
@@ -56,7 +56,7 @@ private void testJsonpException(String wfsVersion)
         JSONType.setJsonpEnabled(false);
 
         // MimeType
-        assertEquals(JSONType.jsonp, response.getContentType());
+        assertEquals(JSONType.JSONP_UTF8, response.getContentType());
```

Comment:
Hmm... I'd go towards having a mime type check facility in the base classes that can ignore the charset.

Reply:
Done